docs/scarpet/api/BlocksAndWorldAccess.md

```diff
@@ -419,6 +419,10 @@ Numeric function, returning the block light at position (from torches and other
 
 Numeric function, returning the sky light at position (from sky access).
 
+### `effective_light (pos)`
+
+Numeric function, returning the "real" light at position. affected by time and weather.
```

Comment:
```suggestion
Numeric function, returning the "real" light at position, which is affected by time and weather.
```
This may be clearer to understand